README.md

```diff
@@ -17,7 +17,7 @@ You just need to register the plugin and you will have response validation enabl
 ```js
 const fastify = require('fastify')()
 
-fastify.register(require('@fastify/response-validation'))
+await fastify.register(require('@fastify/response-validation'))
```

Comment:
we should update the code using the `import` statement by using the top-level await

Reply:
Done in [7d129d4](https://github.com/fastify/fastify-response-validation/pull/71/commits/7d129d4ff9367ba176d4e0636bd2415966d1c68e), please have a look 🙂